ui-ngx/src/app/modules/home/components/widget/lib/gateway/gateway-service-rpc-connector.component.ts

```diff
@@ -424,4 +435,15 @@ export class GatewayServiceRPCConnectorComponent implements OnInit, ControlValue
       this.commandForm.patchValue(value, {onlySelf: false});
     }
   }
+
+  private observeMQTTWithResponse(): void {
+    this.isMQTTWithResponse.valueChanges.pipe(
+      tap((isActive: boolean) => {
+        const responseControl = this.commandForm.get('responseTopicExpression');
+        isActive ? responseControl.enable() : responseControl.disable();
+        responseControl.updateValueAndValidity();
```

Comment:
After enabling or disabling the form control, there is no need to call updateValueAndValidity.